### PR TITLE
Fix code formatting in documentation of dconf.py

### PR DESCRIPTION
**Pull request by bimzhanovt**
##### SUMMARY
Here https://docs.ansible.com/ansible/latest/collections/community/general/dconf_module.html incorrect `:code:` syntax was used to enclose code snippets, which resulted in rendering issues.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
community.general.dconf module

### CODE REVIEW

File: plugins/modules/dconf.py [hunk 1]
```diff
@@ -49,8 +49,8 @@
     I(value="'myvalue'") - with single quotes as part of the Ansible parameter
     value.
   - When using loops in combination with a value like
-    :code:`"[('xkb', 'us'), ('xkb', 'se')]"`, you need to be aware of possible
-    type conversions. Applying a filter :code:`"{{ item.value | string }}"`
+    C("[('xkb', 'us'), ('xkb', 'se')]"), you need to be aware of possible
+    type conversions. Applying a filter C("{{ item.value | string }}")
```
Comment (felixfontein) [inline plugins/modules/dconf.py:53]:
```suggestion
    "[('xkb', 'us'), ('xkb', 'se')]", you need to be aware of possible
    type conversions. Applying a filter "{{ item.value | string }}"
```

**Review by felixfontein — COMMENTED** [review]
Hmm, `C(...)` won't work either since `C(` stops parsing when the next `)` shows up. I think for now it's best to not use formatting (and backport to older versions as well), and start using semantic markup on `main` and `stable-7` (which allows to use `V(...)`) to fix this.

File: plugins/modules/dconf.py (hunk 1 above)
Comment (bimzhanovt) [reply plugins/modules/dconf.py:53]: Should I commit this suggestion?

Comment (felixfontein) [reply plugins/modules/dconf.py:53]: Either that, or add an equivalent change :)

File: plugins/modules/dconf.py [hunk 2]
```diff
@@ -49,8 +49,8 @@
     I(value="'myvalue'") - with single quotes as part of the Ansible parameter
     value.
   - When using loops in combination with a value like
-    :code:`"[('xkb', 'us'), ('xkb', 'se')]"`, you need to be aware of possible
-    type conversions. Applying a filter :code:`"{{ item.value | string }}"`
+    "[('xkb', 'us'), ('xkb', 'se')]", you need to be aware of possible
+    type conversions. Applying a filter "{{ item.value | string }}"
```
Comment (felixfontein) [inline plugins/modules/dconf.py:53]: Ah, wait, sorry, I missed that the second one didn't have a `)` in it... Here `C(...)` is totally fine. Sorry :(
```suggestion
    type conversions. Applying a filter C({{ item.value | string }})
```

Comment (bimzhanovt) [reply plugins/modules/dconf.py:53]: Great, I didn't notice either

**Comment by felixfontein** [comment]
Darn, that was stupid... GitHub is sometimes trying to be too clever :D

**Comment by felixfontein** [comment]
@bimzhanovt thanks a lot for improving the docs!

**Pull request merged**